### PR TITLE
Arch: made HiddenInput component actually hidden

### PR DESCRIPTION
**Pull request by Vultraz**
Also made it used forwardRef. I tested it still works for triggering events (such as in the File field).

### CODE REVIEW

**Comment by MadeByMike** [comment]
Are we sure it wasn't visually hidden only for accessibility reasons? Seems very deliberate.

File: packages/arch/packages/input/src/HiddenInput.js [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
-    ref={innerRef}
+    ref={ref}
     tabIndex="-1"
     css={{
+      display: 'none',
       border: 0,
-      clip: 'rect(1px, 1px, 1px, 1px)',
       height: 1,
       margin: 0,
       opacity: 0,
```
Comment (MadeByMike) [inline packages/arch/packages/input/src/HiddenInput.js:17]: We can remove the tabIndex as well as all the other CSS properties.

**Comment by MadeByMike** [comment]
> Are we sure it wasn't visually hidden only for accessibility reasons? Seems very deliberate.

Scratch this, given the tabindex=-1, it's not accessible anyway.

File: packages/arch/packages/input/src/HiddenInput.js (hunk 1 above)
Comment (Vultraz) [reply packages/arch/packages/input/src/HiddenInput.js]: Removed.

**Pull request merged**